desktop/src/com/limegroup/gnutella/gui/StatusLine.java

```diff
@@ -71,6 +72,11 @@
     private JLabel _bandwidthUsageDown;
     private JLabel _bandwidthUsageUp;
 
+    /**
+     * Shows when bittorrent is disabled due to VPN settings
+     */
+    private VPNBitTorrentDisabledWarningLabel _bittorrent_vpn;
```

Comment:
#style: you've been coding in python lately? ;)
Let's rename to `bittorrentVpn`

luckily in java we have `private`,`public`, `protected` and we don't need to resort to `__mFoo`, `_mBar` naming.